### PR TITLE
Add /organisations endpoint in v1

### PR DESCRIPTION
**Pull request by theseanything**
This PR addresses the following:
- Add organisations endpoint to the /v1 path. 
- Restructured the organisations controller to use ActiveModel serialization.
- Now have the option to specify locales, with english being returned by default.

### CODE REVIEW

File: spec/models/organisation_spec.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+      it 'returns a list of organisations' do
+        organisations = Organisation.find_all
+        expect(organisations).to all be_a(Organisation)
+      end
+
+      it 'returns a list sorted by title' do
+        organisations = Organisation.find_all
+
+        expect(organisations).to match_array([
```
Comment (pmanrubia) [inline spec/models/organisation_spec.rb]: `match_array` ignores the order. So `[1,2,3]` would match `[2,3,1]`. Can you think of a better way to ensure that the order is being kept.

File: spec/models/organisation_spec.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+          have_attributes(name: 'Organisation B'),
+          have_attributes(name: 'Organisation C')
+        ])
+      end
+    end
+
+    context 'locales' do
+      before do
+        create(:edition, document_type: 'organisation', content_id: '1', title: 'English Name', locale: 'en')
+        create(:edition, document_type: 'organisation', content_id: '1', title: 'Enw Cymraeg', locale: 'cy')
```
Comment (pmanrubia) [inline spec/models/organisation_spec.rb]: Shouldn't they have a different `content_id`?

Comment (theseanything) [reply spec/models/organisation_spec.rb]: We infer `organisation id` from the `content_id`. So these tests represent when we have the same organisation (i.e. `organisation_id = 1`) with multiple locales.

File: Gemfile [hunk 3]
```diff
@@ -12,6 +12,7 @@ gem 'plek'
 
 # Third party gems
 gem 'activerecord-import'
+gem 'active_model_serializers', '~> 0.10.0'
```
[the remark this answers is not in the document]
Comment (theseanything) [reply Gemfile]: Removed the version specifier.

File: spec/models/organisation_spec.rb (hunk 1 above)
Comment (theseanything) [reply spec/models/organisation_spec.rb]: Implemented sorted_by matcher.

**Comment by theseanything** [comment]
Split into separate commits.

**Comment by pmanrubia** [comment]
👍

**Pull request merged**